massa-models/src/operation.rs

```diff
@@ -851,21 +801,12 @@ impl WrappedOperation {
             OperationType::Transaction { amount, .. } => *amount,
```

Comment:
Following the doc of this function this should include fee. This function is used to update the balance cache and avoid someone add two transaction that debit him too much during one block.

I think `fee` should be added.

Reply:
I think it's added here: https://github.com/massalabs/massa/pull/3173/files#diff-18062a3d12a9fc5a391c2c894e89f0ecc760f5c5cb8ba0abb9b46ae7f1e09adcR809